data_transfer/copy_gtex_v8_whole_blood_data.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -ex
+
+gsutil -u tx-adapt -m cp "gs://gtex-resources/GTEx_Analysis_v8_QTLs/GTEx_Analysis_v8_EUR_eQTL_all_associations/Whole_Blood.v8.EUR.allpairs.*" gs://cpg-gtex-main/v8/whole_blood/
```

Comment:
Did you also want to copy a subset of the data (e.g. chr22) to the `test` bucket?

Reply:
Good catch. Yes, I've now added that in.